src/components/AdminSettings.vue

```diff
@@ -14,6 +14,16 @@
 				{{ t('richdocuments', 'Collabora Online is a powerful LibreOffice-based online office suite with collaborative editing, which supports all major documents, spreadsheet and presentation file formats and works together with all modern browsers.') }}
 			</p>
 
+			<!-- New Collabora Admin Settings Section -->
+			<div id="admin-cool-frame-section">
```

Comment:
Please move this further down, this is certainly less important then configuring the URL.

Also before merging we should make sure it is hidden if the collabora server does not provide the ability to show the iframe:

<img width="1307" alt="Screenshot 2025-01-31 at 17 15 04" src="https://github.com/user-attachments/assets/aece2da0-e8c6-499c-91b0-199f87b8fd37" />